modules/accounting/model/accounting.class.php

```diff
@@ -2365,7 +2365,7 @@ public function getAccountPlanFromOrgNumber($OrgNumber) {
 
 
     // Check if JournalID exists with a given VoucherType
-    public function checkJournalID($VoucherType, $JournalID) {
+    public function checkJournalID($VoucherType, $JournalID, $Period) {
```

Comment:
SELECT JournalID, COUNT( * ) 
FROM accountline
WHERE Period =  '2016-11'
AND Active =1
GROUP BY JournalID